### DISCUSSION TITLE
tailwindcss opacity-0 property not working

### DISCUSSION THREAD

**Discussion by treconyl** [tailwindlabs/tailwindcss#11053] [Help]
there seems to be some problem making tailwindcss opacity-0 property not working properly, i have switched to using invisible and group-hover:visible instead in my project. So is this a bug?
![image](https://user-images.githubusercontent.com/75132211/233540640-6d8ca6fe-2612-4ce1-a91a-a5b5be082577.png)

**Comment by wongjn** [comment]
Consider checking:
- The rule is coming from Tailwind.
- Your Tailwind configuration. It could be that your final resolved configuration is setting `theme.opacity.0` to be `'hover'` for some reason.
- Any custom CSS that might have this rule.

**Reply by treconyl** [reply]
my tailwind config is empty, without any changes, theme.extend.., thanks for the advice 👍

**Reply by wongjn** [reply]
Consider providing a remotely-hosted git repository that reproduces the unexpected behavior if you'd like us to take a closer look.
